dessia_common/schemas/core.py

```diff
@@ -598,11 +637,15 @@ def serialized(self) -> str:
         """ Full class name. """
         return full_classname(object_=self.annotation, compute_for='class')
 
+    @property
+    def standalone_in_db(self) -> bool:
+        """ Whether the class is standalone in db. """
+        return self.annotation._standalone_in_db
+
     def to_dict(self, title: str = "", editable: bool = False, description: str = ""):
         """ Write CustomClass as a Dict. """
         chunk = super().to_dict(title=title, editable=editable, description=description)
-        chunk.update({'type': 'object', 'standalone_in_db': self.annotation._standalone_in_db,
-                      "classes": [self.serialized]})
+        chunk.update({'type': 'object', 'standalone_in_db': self.standalone_in_db, "classes": [self.serialized]})
```

Comment:
Are you sure you want to compute twice (one wrong and one good) attributes standalone_in_db and serialized (especially serialized) ?

Reply:
After some thoughts, i guess you compute it only once. But why inheriting from Schemas when you already have a class Class that inherits from Schemas and you could use as parent ?